platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/SwirldsPlatform.java

```diff
@@ -460,24 +470,53 @@ public class SwirldsPlatform implements Platform {
 
         components.add(new IssMetrics(platformContext.getMetrics(), currentAddressBook));
 
+        // Manages the pipeline of signed states to be written to disk
+        final SignedStateFileManager signedStateFileManager = new SignedStateFileManager(
+                platformContext,
+                new SignedStateMetrics(platformContext.getMetrics()),
+                Time.getCurrent(),
+                actualMainClassName,
+                selfId,
+                swirldName);
+        // FUTURE WORK: at some point this should be part of the unified platform wiring
+        final WiringModel model = WiringModel.create(platformContext, Time.getCurrent());
+        final TaskScheduler<StateSavingResult> savedStateScheduler = model.schedulerBuilder("signed_state_file_manager")
+                .withType(TaskSchedulerType.SEQUENTIAL_THREAD)
+                .withUnhandledTaskCapacity(stateConfig.stateSavingQueueSize())
+                .withExternalBackPressure(false)
+                .build()
+                .cast();
+        final SignedStateFileManagerWiring signedStateFileManagerWiring =
+                new SignedStateFileManagerWiring(savedStateScheduler);
+        signedStateFileManagerWiring.bind(signedStateFileManager);
+        signedStateFileManagerWiring.solderPces(preconsensusEventWriter);
+        signedStateFileManagerWiring.solderStatusManager(platformStatusManager::submitStatusAction);
+        signedStateFileManagerWiring.solderAppCommunication(appCommunicationComponent::stateSavedToDisk);
```

Comment:
Could we instead use the wiring model inside `PlatformWiring.java`? It seems odd to have multiple wiring model instances for the same platform.

Reply:
```
// FUTURE WORK: at some point this should be part of the unified platform wiring
```

Ah, just saw your FUTURE WORK message. It's ok if we use a stand alone wiring model while waiting for Austin's PR to merge (since messing with `PlatformWiring.java` is likely to be a source of merge conflicts).

Reply:
yes, exactly why I didn't want to do it in this PR